commercetools-models/src/main/java/io/sphere/sdk/products/search/ProductCategoriesIdTermFacetSearchModel.java

```diff
@@ -24,6 +24,9 @@
     @Override
     ProductCategoriesIdTermFacetSearchModel<T> withAlias(final String alias);
 
+    @Override
+    ProductCategoriesIdTermFacetSearchModel<T> countingProducts(final Boolean isCountingProducts);
```

Comment:
Following the structure of `alias` I would call it `withCountingProducts`. It is consistent with the semantics throughout the SDK, as it is returning an object with the same characteristics as the current one but with the counting products to whatever value you gave. The other methods are moving forward instead.